modules/apps/headless/headless-admin-content/headless-admin-content-impl/src/main/java/com/liferay/headless/admin/content/internal/resource/v1_0/StructuredContentResourceImpl.java

```diff
@@ -1,11 +1,11 @@
 /**
  * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
- * <p>
+ *
```

Comment:
Now it is the other way around? I think source formatter is making something a little estrange here...

Reply:
Source formatter prefer without `<p>` in the copyright. Is my fault due to a shortcut I use with IJ to refactor the code. I have to change the configuration to avoid that.